x-pack/plugins/observability_solution/dataset_quality/public/components/dataset_quality/filters/selector.tsx

```diff
@@ -31,6 +32,7 @@ export interface Item {
 }
 
 export function Selector({
+  dataTestSubj = 'datasetQualitySelectable',
```

Comment:
Is the `Selector` used from a component other than `Filters`? If not, the default value could be skipped and the prop can be made required